### PR TITLE
Snapshot and throughput flow tests

### PR DESCRIPTION
**Pull request by pkoryzna**

### CODE REVIEW

File: monitoring/src/test/scala/akkaviz/events/LightSnapshotTest.scala [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+import akka.testkit.{TestKit, TestActorRef}
+import akkaviz.events.types._
+import org.scalatest.{FunSuiteLike, Matchers}
+
+class LightSnapshotTest() extends TestKit(ActorSystem("SnapshotTests")) with FunSuiteLike with Matchers with Helpers {
+
+  val firstRef = TestActorRef[SomeActor](new SomeActor, "first")
+  val secondRef = TestActorRef[SomeActor](new SomeActor, "second")
+
+  test("should include actors receiving messages as live") {
```
Comment (JJag) [inline monitoring/src/test/scala/akkaviz/events/LightSnapshotTest.scala]: typo alive*

**Comment by JJag** [comment]
:cool:

File: monitoring/src/test/scala/akkaviz/events/ThroughputMeasurementFlowTest.scala [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+      whenReady(materialized) { r =>
+        r should be('empty)
+      }
+    }
+
+    "emit proper measured value for one message" in {
+      val src = Source.single(ReceivedWithId(1, ActorRef.noSender, firstRef, "sup", true))
+      val mat = src.via(ThroughputMeasurementFlow(1.second))
+        .toMat(Sink.head[ThroughputMeasurement])(Keep.right).run()
```
Comment (lustefaniak) [inline monitoring/src/test/scala/akkaviz/events/ThroughputMeasurementFlowTest.scala]: Codacy Issue found: [Usage of head on collections.](https://www.codacy.com/app/lustefaniak/akka-viz/file/2355072435/issues/source?bid=3121282&fileBranchId=3217410#l42)

File: monitoring/src/test/scala/akkaviz/events/ThroughputMeasurementFlowTest.scala [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+
+  override implicit val patienceConfig = PatienceConfig(timeout = 5.seconds)
+
+  "ThroughputMeasurementFlow" should {
+
+    "not emit any measurements if there are no Received events" in {
+      val src = Source.empty[BackendEvent]
+      val sink: Sink[BackendEvent, Future[List[BackendEvent]]] = Sink.fold(List.empty[BackendEvent])((list, ev) => ev :: list)
+
+      val materialized = ThroughputMeasurementFlow(1.second).runWith(src, sink)._2
```
Comment (lustefaniak) [inline monitoring/src/test/scala/akkaviz/events/ThroughputMeasurementFlowTest.scala]: Codacy Issue found: [Found unnamed result or parameter](https://www.codacy.com/app/lustefaniak/akka-viz/file/2355072435/issues/source?bid=3121282&fileBranchId=3217410#l32)

**Pull request merged**